test/fixture/index_hookConsole.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const electron = require('electron');
+const logger = require('../..');
+
+let mainWindow;
+electron.app.on('ready', async () => {
+  mainWindow = new electron.BrowserWindow();
```

Comment:
Should use tab indentation. XO ignores fixture directories by default, which is why it was not caught by the linter.